opensearch/src/test/java/org/opensearch/sql/opensearch/OpenSearchTestBase.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch;
+
+import org.opensearch.sql.expression.DSL;
+import org.opensearch.sql.expression.config.ExpressionConfig;
+
+public class OpenSearchTestBase {
+
+  protected static DSL dsl;
+
+  {
+    ExpressionConfig config = new ExpressionConfig();
+    dsl = config.dsl(config.functionRepository(config.functionExecutionContext()));
+  }
```

Comment:
Can you make `dsl` `Autowired`?

Reply:
If you name variable `DSL` (uppercase) you can avoid hundreds changes in inherited test classes.

Reply:
> If you name variable `DSL` (uppercase) you can avoid hundreds changes in inherited test classes.

`namedArgument` is a static method -- it works fine without an instance of `DSL` class.  With this updated it's clearer where `DSL` instance is required and therefore  which methods need Spring.